examples/django/README.md

```diff
@@ -0,0 +1,55 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
+
+2. You must set both project_id & project_key in `settings.py`.
```

Comment:
I am a Django noob, so I had to search for this `settings.py`. Let's change that to `weather/settings.py`.